src/test/java/web/tests/SortingFilter.java

```diff
@@ -0,0 +1,63 @@
+package web.tests;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class SortingFilter {
```

Comment:
Instead of creating a new class, define the sorting logic inside the FlipkartTest class